tests/unit/session/test_provision.py

```diff
@@ -27,14 +27,21 @@ def next_tox_major():
     return "10.0.0"
 
 
-def test_provision_min_version_is_requires(newconfig, next_tox_major):
+@pytest.fixture(scope="session", params=["minversion", "min_version"])
+def minversion_option(request):
+    """both possible names for the minversion config option"""
+    return request.param
+
+
+def test_provision_min_version_is_requires(newconfig, minversion_option, next_tox_major):
     with pytest.raises(MissingRequirement) as context:
         newconfig(
             [],
             """\
             [tox]
-            minversion = {}
+            {} = {}
```

Comment:
I wonder if adding a new fixture that evaluates to this expression is worth it or not.